test/functional/conftest.py

```diff
@@ -291,11 +299,8 @@ def supports_docker():
 
 @pytest.helpers.register
 def supports_linode():
-    try:
-        pytest.importorskip('linode')
-        return True
-    except Exception:
-        return False
+    # FIXME: Travis CI
+    return not IS_TRAVIS and get_lxd_executable()
```

Comment:
`%s/get_lxd_executable/get_linode_executable/`

Reply:
oh... I missed that. btw you can suggest changes in PRs with `Ctrl+G`:
```suggestion
    return not IS_TRAVIS and get_linode_executable()
```